eventmesh-storage-plugin/eventmesh-storage-mongodb/src/main/java/org/apache/eventmesh/storage/mongodb/utils/MongodbCloudEventUtil.java

```diff
@@ -29,10 +29,12 @@
 import io.cloudevents.SpecVersion;
 import io.cloudevents.core.builder.CloudEventBuilder;
 
+import static org.apache.eventmesh.storage.mongodb.constant.MongodbConstants.*;
```

Comment:
checkstyle doesn't seem to support '. *' import.

Reply:
Thanks for the feedback, 

Updated in my last commit.